app/actors/curation_concerns/actors/add_as_member_of_collections_actor.rb

```diff
@@ -0,0 +1,23 @@
+module CurationConcerns
+  module Actors
+    class AddAsMemberOfCollectionsActor < AbstractActor
+      def create(attributes)
+        collection_ids = attributes.delete(:member_of_collection_ids)
+        next_actor.create(attributes) && add_to_collections(collection_ids)
+      end
+
+      def update(attributes)
+        collection_ids = attributes.delete(:member_of_collection_ids)
+        add_to_collections(collection_ids) && next_actor.update(attributes)
```

Comment:
I was following the pattern of the existing AddToWorkActor and AddToCollectionActor — I think the rationale for switching them might be to make sure that the Work is created before trying to add it to the Collections.  But that doesn't apply here, so I'll make them both add to Collections and then call the next Actor on the stack.